publisher/publisher.go

```diff
@@ -111,6 +111,7 @@ func NewLog(uri, b64PK string, logger blog.Logger) (*Log, error) {
 	opts := jsonclient.Options{
 		Logger:    logAdaptor{logger},
 		PublicKey: pemPK,
+		UserAgent: "boulder/1.0",
```

Comment:
I think this should be configurable similar to the `VA's user agent. We know other organizations are running Boulder and we'd probably want to encourage them to deploy their instances with a UA distinct from ours.

It also seems like writing a quick unit test that the correct UA ends up being seen by the remote http server wouldn't be too hard. WDYT?